torch/_dynamo/variables/builtin.py

```diff
@@ -805,8 +805,10 @@ def call_dict_helper(tx, user_cls, arg, **options):
         else:
             raise AssertionError("call_dict_helper with illegal arg")
 
-    def call_dict(self, tx, obj=None):
+    def call_dict(self, tx, obj=None, **kwargs):
         if self.is_supported_call_dict_arg(tx, obj):
+            if kwargs:
+                return self.call_dict_helper(tx, dict, variables.ConstDictVariable(kwargs, dict))
```

Comment:
* We should check if ```kwargs``` is supported, you can extend the function ```is_supported_call_dict_arg``` to ```is_supported_call_dict_arg_kwargs```. For unsupported kwargs, we just fall back. In your current implementation, it throws exception for unsupported cases.
* When you building ```variables.ConstDictVariable(kwargs, dict)```, you have to fill the ```source``` and ```mutable_local```, which is used when reconstruction after graph break.

Reply:
1. What is an example of unsupported kwargs we should fall back on?

2. `call_dict_helper` when called with a `ConstDictVariable` as the `arg` will set `mutable_local`. However, `source` is not set for any `ConstDictVariable` in this file, so I'm not sure what to use for it.